src/Passport.php

```diff
@@ -36,6 +36,13 @@ class Passport
         //
     ];
 
+    /**
+     * The authorization server response formatter.
+     *
+     * @var \League\OAuth2\Server\ResponseTypes\ResponseTypeInterface|null
+     */
+    public static $authorizationServerResponseType = null;
```

Comment:
Think you can leave off the `null` here?
```suggestion
    public static $authorizationServerResponseType;
```

Reply:
Sure :100: